src/core/js/startController.js

```diff
@@ -70,7 +70,7 @@ define([
                         if (model.get("_isComplete")) continue;
                     }
 
-                    if (!className || $("html").is(className)) {
+                    if (!className || $("html").is('.' + className)) {
```

Comment:
I know the previous implementation was incorrect but this could be made backwards compatible if we did something along the lines of:
```js
var $html = $("html");

if (!className || $html.is(className) || $html.hasClass(className)) {